src/app/shared/components/project-title/project-title.component.ts

```diff
@@ -1,32 +1,39 @@
 // Copyright The Linux Foundation and each contributor to CommunityBridge.
 // SPDX-License-Identifier: MIT
 
-import { Component, OnInit, Input, EventEmitter, Output } from '@angular/core';
+import { Component, OnInit, Input, EventEmitter, Output, ViewChild, TemplateRef } from '@angular/core';
 import { AlertService } from '../../services/alert.service';
 import { ClaContributorService } from 'src/app/core/services/cla-contributor.service';
 import { ProjectModel } from 'src/app/core/models/project';
-import { UserModel } from 'src/app/core/models/user';
+import { UserFromSessionModel, UserFromTokenModel } from 'src/app/core/models/user';
```

Comment:
Similar here - I think you need to add two new models but don't remove the already exiting one...